INSTALL.md

```diff
@@ -130,10 +130,10 @@ Some colours, SVGs and other files are generated with helper scripts. Not all us
 
 For deployment, CartoCSS and Mapnik are required.
 
-* [CartoCSS](https://github.com/mapbox/carto) >= 0.18.0 (we're using YAML)
-* [Mapnik](https://github.com/mapnik/mapnik/wiki/Mapnik-Installation) >= 3.0
+* [CartoCSS](https://github.com/mapbox/carto) >= 1.2.0 (we're using YAML)
+* [Mapnik](https://github.com/mapnik/mapnik/wiki/Mapnik-Installation) >= 3.0.22
```

Comment:
Just flagging that this isn't on the current Ubuntu LTS, and is likely to be a headache.

Reply:
Yes. On the other hand: It will be part of Ubuntu 20.04 LTS, which will be available in only two months. There is also a backport to Ubuntu 18.04 LTS available, provided by @tomhughes [in a PPA](https://github.com/gravitystorm/openstreetmap-carto/pull/3399#issuecomment-519016170) and [yet deployed and running on openstreetmap.org servers](https://github.com/gravitystorm/openstreetmap-carto/pull/3399#issuecomment-520025952). So for openstreetmap.org this would make no difference. But of course for other downstream projects this might be different. I will comment this in INSTALL.MD.

Could we bump Mapnik und CartoCSS versions together with the schema changes to avoid two mayer releases in short time?

Reply:
Is available now since more than two years: has been part of Ubuntu since 20.04 LTS.